src/components/movie-view/movie-view.jsx

```diff
@@ -1,26 +1,24 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-const MovieView = ({ movie, onBackClick }) => {
+import { Card, Button } from 'react-bootstrap';
+import { useParams, useNavigate } from 'react-router-dom';
```

Comment:
You're using `react-router-dom` here as required, did you install it and how did you do it?

Reply:
i used command promt